### ISSUE TITLE
Loading checkpoint model (custom layer issue)

### ISSUE THREAD

**Issue by mrtgenet** [pierluigiferrari/ssd_keras#235]
Hello,

First, **thank you** for your your useful tutorials ! 

I am trying to train an ssd512, following the ssd300.ipynb tutorial and changing what needed to be changed (such as predictor_sizes and stuff) and all works fine. Yet I met a problem when I first needed to _resume training from a checkpoint_ (full model saved as ssd512_epoch-X_loss-Y_val_loss-Z.h5). I cannot load it because of **custom layers** issue. I checked issue 18 (https://github.com/pierluigiferrari/ssd_keras/issues/18) where it seems like things went fine and load_model should work, yet I stell get errors.

Since the concerned imports are 
`from keras_loss_function.keras_ssd_loss import SSDLoss
from keras_layers.keras_layer_AnchorBoxes import AnchorBoxes
from keras_layers.keras_layer_DecodeDetections import DecodeDetections
from keras_layers.keras_layer_DecodeDetectionsFast import DecodeDetectionsFast
from keras_layers.keras_layer_L2Normalization import L2Normalization`

...I tried to load the model with 
`custom_objects = {'L2Normalization': L2Normalization,
                  'AnchorBoxes': AnchorBoxes,
                  'DecodeDetections': DecodeDetections,
                  'DecodeDetectionsFast': DecodeDetectionsFast,
                  'SSLoss': SSDLoss}`

...which raised the error "**ValueError**: Unknown loss function:compute_loss"

So I tried adding "'compute_loss': SSDLoss.compute_loss" in custom_objects, which instead raised the error "**TypeError**: compute_loss() missing 1 required positional argument: 'y_pred'"

**Loading those full model checkpoints with the optimizer state, which if I'm right is kind of important to correctly resume training, seems to be not so straight forward. 
Could I please get some help about this issue ? Also maybe this should be included to the the tutorials ?**

Thanks :)
